content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -98,6 +106,39 @@ data:
 EOF
 {{< /text >}}
 
+For clusters that use CoreDNS:
+{{< text bash >}}
```

Comment:
@danehans - I gave your coredns configuration a spin.  Unfortunately coredns does not appear to resolve (foo.bar2).  Instead the FQDN must be specified (foo.bar2.global).

E.g.:
```
/ # nslookup httpbin.bar2
nslookup: can't resolve '(null)': Name does not resolve

nslookup: can't resolve 'httpbin.bar2': Name does not resolve
/ # nslookup httpbin.bar2.global
nslookup: can't resolve '(null)': Name does not resolve

Name:      httpbin.bar2.global
Address 1: 127.255.0.3
/ # nslookup httpbin.default.global
nslookup: can't resolve '(null)': Name does not resolve
```

I spent a couple hours working on the config with different plugins to see if I could get this to work without a `.global` suffix in DNS properly resovling the application IP.  I wasn't successful, but coredns has pretty rich functionality, so it may be viable.

@frankbu @rshriram any ideas?